rubocop.gemspec

```diff
@@ -32,7 +32,7 @@ Gem::Specification.new do |s|
   }
 
   s.add_dependency('json', '~> 2.3')
-  s.add_dependency('language_server-protocol', '>= 3.17.0')
+  s.add_dependency('language_server-protocol', '~> 3.17.0.2')
```

Comment:
Probably this also deserves a comment here, given that such version numbers are uncommon in the wild.

Reply:
Indeed! I added an explanation comment.